crates/astria-core/src/protocol/transaction/v1alpha1/action.rs

```diff
@@ -252,6 +260,30 @@ impl TryFrom<raw::Action> for Action {
     }
 }
 
+pub(super) trait ActionName {
```

Comment:
Please remove this. All of these should be done via the `Protobuf` trait implemented at the root of the crate - I intend to rename it to `DomainType`.

Reply:
Discussed offline: this should flow through `Protobuf::full_name` but is out of scope for this PR. @Lilyjjo will create a PR to follow up on this.

Reply:
Tracked in a note in the code and here: #1567